containers/zammad/docker-entrypoint.sh

```diff
@@ -27,6 +27,8 @@ set -e
 : "${ZAMMAD_RAILSSERVER_PORT:=3000}"
 : "${ZAMMAD_WEBSOCKET_HOST:=zammad-websocket}"
 : "${ZAMMAD_WEBSOCKET_PORT:=6042}"
+: "${ZAMMAD_WEB_CONCURRENCY:=0}"
+: "${ZAMMAD_SESSION_JOBS_CONCURRENT:=}"
```

Comment:
ZAMMAD_SESSION_JOBS_CONCURRENT has no default value set?

Reply:
Yes. The ENV by default is empty

Reply:
If its empty is it needed?
We could just use it from the docker compose file like here: 
https://github.com/zammad/zammad-docker-compose/blob/master/docker-compose.yml#L11

Reply:
My plan was to use it via the `.env` file which personally would be my way to go. 
Not sure if above is a bad intention of mine.

Any way, I'm struggling with the puma part right now any way so... :-D